README.md

```diff
@@ -21,19 +21,54 @@ see https://stackoverflow.com/questions/71262775/how-do-i-ensure-assets-are-pres
   - postgres
   - yarn
 
-You can install postgres with `brew install postgresql` if you have homebrew. Then please run `bundle exec rails db:create` to set up the development and test databases.
+## Setting up the app
 
-Install dependencies:
+You can use [Homebrew](https://brew.sh/) to install any dependencies. The Homebrew documentation has lots of useful commands.
+
+If using Homebrew:
+
+Install postgres. You will need to select a version if using brew. Run the below command, changing the version number as appropriate.
+
+```
+brew install postgresql@14
+```
+
+You will be prompted on the command line to start the server with something like:
 
 ```
+brew services start postgresql@14
+```
+
+Install [Bundler](https://bundler.io/). You may run into an error where you need to update your Ruby version. Use rbenv or another version manager to do this.
+
+```
+gem install bundler
 bundle install
+```
+
+Create the development and test databases
+
+```
+bundle exec rails db:create
```

Comment:
You will also need to have the databases loaded up with the correct table structure, via either `bundle exec rails db:migrate` (to apply the migrations one by one), or `bundle exec rails db:schema:load` (to load the schema file). That's probably worth noting (and apologies, I was the one who added a database to the app and didn't update the readme properly)

Reply:
I think I might have done a `db:reset` now I think about it but I will document the way you've suggested instead!